web_console/web-console.adoc

```diff
@@ -21,3 +21,8 @@ Platform 4.x Tested Integrations] page before you create the supporting
 infrastructure for your cluster.
 
 include::modules/web-console-overview.adoc[leveloffset=+1]
+include::modules/multi-cluster-about.adoc[leveloffset=+1]
+include::modules/enabling-multi-cluster-console.adoc[leveloffset=+2]
+
+.Additional resources
+* xref:../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling feature sets using the web console]
```

Comment:
missing anchor ID in file path https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references